server/engine/src/main/java/org/eclipse/lsp/cobol/service/AnalysisService.java

```diff
@@ -194,7 +194,6 @@ public void reanalyzeDocument(String uri, String text) {
   public void stopAnalysis(String uri) {
     CobolDocumentModel documentModel = documentService.get(uri);
 
-    contentCache.invalidate(uri);
```

Comment:
Don't invalidate cache, as we need the updated content to update the CopybookModel https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/eb0c674468d09b505ec3711a25126c8c73412a5c/server/engine/src/main/java/org/eclipse/lsp/cobol/service/copybooks/CopybookServiceImpl.java#L128